src/pages/signin/LoginForm.js

```diff
@@ -77,6 +77,7 @@ class LoginFormNarrow extends React.Component {
                         autoCapitalize="none"
                         placeholder="Phone or Email"
                         placeholderTextColor={themeColors.placeholderText}
+                        autofocus
```

Comment:
Curious if there was some reason we were not using `autofocus` before in the `LoginFormNarrow`? I feel like the answer is: yes, since we want to first show the marketing stuff (image and welcome text) before popping the keyboard over everything.

I also noticed that the input does not actually auto focus on small screens.

Reply:
ah, I'll do this conditionally based on the screen width